packages/dashboards/addon/templates/dashboards/dashboard/widgets/widget.hbs

```diff
@@ -9,9 +9,15 @@
     {{/if}}
     {{navi-icon "angle-right"}}
     {{#with (model-for "dashboards.dashboard") as |parentDashboard|}}
-      {{#link-to "dashboards.dashboard" parentDashboard.id class="navi-report-widget__breadcrumb-link"}}
-        {{parentDashboard.title}}
-      {{/link-to}}
+      {{#if parentQueryParams}}
+        {{#link-to "dashboards.dashboard" parentDashboard.id (query-params filters=parentQueryParams.filters) class="navi-report-widget__breadcrumb-link"}}
```

Comment:
Would it work to move the `if` inside the `link-to` helper to just wrap the `query-params`?

Reply:
no, ember 2.x bad.